gateway/ln-gateway/src/endpoint_constants.rs

```diff
@@ -0,0 +1,13 @@
+pub const PAY_INVOICE: &str = "pay_invoice";
```

Comment:
nit: would prefer `ENDPOINT` on the end of each of these, like we do for fedimint server endpoints